### DISCUSSION TITLE
umi ui has no image show in localhost:3000

### DISCUSSION THREAD

**Discussion by cmdparkour** [umijs/umi#6421] [Q&A] [answered]
when I open it, I can't see any pictures in the localhost:3000, but i can see the name of the block or template, I'm from China.
当我打开localhost:3000时，我无法看到任何的图片，但是我可以看到block或者是template的名称，但是就是没有图片。

**Comment by xiaohuoni** [comment, marked as answer]
【前端情报】😘 让你“爱”上 GitHub，解决访问时图裂、加载慢的问题。（无需安装）
https://github.com/521xueweihan/GitHub520
